indispensable-josef/src/jvmTest/kotlin/at/asitplus/signum/indispensable/josef/JweEncryptedTest.kt

```diff
@@ -1,31 +1,42 @@
 package at.asitplus.signum.indispensable.josef
 
+import at.asitplus.signum.HazardousMaterials
 import at.asitplus.signum.indispensable.io.Base64UrlStrict
 import at.asitplus.signum.indispensable.symmetric.randomKey
 import at.asitplus.signum.indispensable.toCryptoPublicKey
 import at.asitplus.signum.supreme.symmetric.decrypt
 import at.asitplus.signum.supreme.symmetric.encrypt
+import at.asitplus.testballoon.invoke
+import at.asitplus.testballoon.minus
+import at.asitplus.testballoon.withData
 import com.nimbusds.jose.*
 import com.nimbusds.jose.crypto.AESEncrypter
 import com.nimbusds.jose.crypto.ECDHEncrypter
 import com.nimbusds.jose.jwk.Curve
 import com.nimbusds.jose.jwk.gen.ECKeyGenerator
 import com.nimbusds.jose.util.Base64URL
-import io.kotest.core.spec.style.FreeSpec
-import io.kotest.datatest.withData
-import io.kotest.matchers.nulls.shouldNotBeNull
+import de.infix.testBalloon.framework.testSuite
 import io.kotest.matchers.shouldBe
 import io.matthewnelson.encoding.base16.Base16
 import io.matthewnelson.encoding.core.Encoder.Companion.encodeToString
+import org.kotlincrypto.random.CryptoRand
+import org.kotlincrypto.random.DelicateCryptoRandApi
 import java.net.URI
 import javax.crypto.KeyGenerator
 import kotlin.random.Random
 
 
-class JweEncryptedTest : FreeSpec({
+@OptIn(DelicateCryptoRandApi::class)
+private object InsecureRandom : CryptoRand() {
+    override fun nextBytes(buf: ByteArray) = Random.nextBytes(buf)
+    fun nextBytes(n: Int) = ByteArray(n).also { nextBytes(it) }
+}
```

Comment:
I am unsure what the point of using this RNG in tests is? Isn't this equivalent to whatever `Random.Default` is doing?

Reply:
CryptRandom does intentionally not implement Random JS targets run out of entropy shockingly fast, so we need this

Reply:
done